docker/scripts/README.md

```diff
@@ -17,10 +17,8 @@ Don't forget the
 ## Install nvidia-docker (optional)
 
 To use the host machine's GPUs, you need to install nvidia docker.
+Please follow https://github.com/NVIDIA/nvidia-docker to install the latest version.
```

Comment:
The usage of nvidia-docker2 was discourged in favor of nvidia-container-runtime.
See docs/specs/prerequisite_software_installation_guide.md for more.